pkg/core/mempool/mem_pool.go

```diff
@@ -154,13 +160,25 @@ func (mp *Pool) Add(t *transaction.Transaction, fee Feer) error {
 		mp.lock.Unlock()
 		return ErrDup
 	}
-	err := mp.checkTxConflicts(t, fee)
+	err := mp.checkAttributesConflicts(t, fee)
+	if err != nil {
+		mp.lock.Unlock()
+		return err
+	}
+	err = mp.checkTxConflicts(t, fee)
```

Comment:
And BTW, it's used by `Verify` and it should also take P2PSigExtensions into account.